demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -717,22 +891,24 @@ def build_dependency_graph(pack_id, id_set):
         return graph
 
     @staticmethod
-    def find_dependencies(pack_name, id_set_path=None):
+    def find_dependencies(pack_name, id_set_path=None, debug_file_path=None):
         """
         Main function for dependencies search and pack metadata update.
 
         Args:
             pack_name (str): pack id, currently pack folder name is in use.
             id_set_path (str): id set json.
+            debug_file_path (str): path to dependency explanations file.
```

Comment:
Doesn't mypy return error here?
id_set_path, debug_file_path are of type `str` while their default value is None.
Consider changing their default to `''`